client/src/components/StandardFAQ/StandardFAQ.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { SomeThingsToKeepInMind } from "src/panels/panel_declarations/common_panel_components";
+
+import { KeyConceptList } from "src/components/KeyConceptList/KeyConceptList";
+
+import { secondaryColor } from "src/style_constants/colors.interop.scss";
+
+type StandardFAQProps = {
+  faq_content: React.ReactNode[][]; // TODO: change to [React.ReactNode, React.ReactNode][] when faq_utils gets converted to ts
+  is_initially_expanded?: boolean;
+  background_color?: string;
+} & typeof StandardFAQ.defaultProps;
+
+export class StandardFAQ extends React.Component<StandardFAQProps> {
```

Comment:
Hmm, "standard" is a bit vague. Maybe name this `PinnedFAQ` or even `PinnedFAQDrawer`?